_data/events.yaml

```diff
@@ -2,18 +2,15 @@ items:
   - name: 'Field of Study: Data Science and Women in Data Science'
```

Comment:
I wonder if maybe this file could also link to the program page of the program that's running it? either URL or whatever ID it stores them as?

(Just general thinking, maybe don't need to do it before the demo)

Reply:
It doesn't have to be perfect either. We can always add in supporting details in our E&I final doc.